libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -151,8 +151,9 @@ singleton AP_Vehicle method get_time_flying_ms uint32_t
 
 include AP_SerialLED/AP_SerialLED.h
 singleton AP_SerialLED alias serialLED
-singleton AP_SerialLED method set_num_LEDs boolean uint8_t 1 16 uint8_t 0 32
-singleton AP_SerialLED method set_RGB void uint8_t 1 16 uint32_t 0U UINT32_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
+singleton AP_SerialLED method set_num_LEDs boolean uint8_t 1 16 uint8_t 0 64
```

Comment:
Ideally the limit of 64 is either a define, or a public static const member of the AP_SerialLED interface and we could then refer to it by name here, which would be much more flexibile if we ever updated the number of LED's supported.

Reply:
fixed